.ci/deploy/pantheon/dev-multidev

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+set -eo pipefail
+
+#
+# This script deploys the build artifact to Pantheon.
+# On the master branch the dev environment is used.
+# Otherwise a multidev environment is used.
+#
+
+# Authenticate with Terminus
+terminus -n auth:login --machine-token="$TERMINUS_TOKEN"
+
+# Prepare for Pantheon
+composer run prepare-for-pantheon
+
+if [[ $CI_BRANCH != $DEFAULT_BRANCH ]]
+then
+  # Create a new multidev environment (or push to an existing one)
+  terminus -n build:env:create "$TERMINUS_SITE.dev" "$TERMINUS_ENV" --yes
+else
+  # Push to the dev environment
+  terminus -n build:env:push "$TERMINUS_SITE.dev" --yes
+fi
+
+# Update the Drupal database
+terminus -n drush "$TERMINUS_SITE.$TERMINUS_ENV" -- updatedb -y
+
+# If exported configuration is available, then import it.
+if [ -f "config/system.site.yml" ] ; then
+  terminus -n drush "$TERMINUS_SITE.$TERMINUS_ENV" -- config-import --yes
+fi
+
+# Clear Drupal cache
+terminus -n drush "$TERMINUS_SITE.$TERMINUS_ENV" -- cr
+
+# Clear the environment cache
+terminus -n env:clear-cache $TERMINUS_SITE.$TERMINUS_ENV
```

Comment:
Doesn't this call `drush cr` as part of its workflow?

Reply:
I'm not sure. The [Terminus source just shows the clear cache workflow being called](https://github.com/pantheon-systems/terminus/blob/fcbfecb8b74926e9539f2eca87643618d6256338/src/Models/Environment.php#L144).

If you're confident the clear cache workflow clears Drush cache, and not just Varnish, we can remove the `drush cr` call.

However, I don't think to leave it in hurts in case the workflow behavior ever changes.